proposals/improved-interpolated-strings.md

```diff
@@ -287,12 +270,107 @@ Do we want to have builders for `System.IFormattable` and `System.FormattableStr
 
 _Answer_: No.
 
-### Lowering
+### Builder pattern codegen
+
+In this section, method invocation resolution refers to the steps listed [here](https://github.com/dotnet/csharplang/blob/main/spec/expressions.md#method-invocations).
+
+#### `Create` method invocation resolution
```

Comment:
Have we got the `Create` vs. constructor discussin on the LDM agenda?

Reply:
That's one of the 2 or 3 questions left, which I hope to fully cover in the next session.